bazel/toolchains/gcc_aarch64_cross_toolchain_config.bzl

```diff
@@ -0,0 +1,363 @@
+        name = "user_compile_flags",
+        enabled = True,
+        flag_sets = [
+            flag_set(
+                actions = [
+                    ACTION_NAMES.assemble,
+                    ACTION_NAMES.preprocess_assemble,
+                    ACTION_NAMES.linkstamp_compile,
+                    ACTION_NAMES.c_compile,
+                    ACTION_NAMES.cpp_compile,
+                    ACTION_NAMES.cpp_header_parsing,
+                    ACTION_NAMES.cpp_module_compile,
+                    ACTION_NAMES.cpp_module_codegen,
+                    ACTION_NAMES.lto_backend,
+                    ACTION_NAMES.clif_match,
+                ],
+                flag_groups = [
+                    flag_group(
+                        flags = ["%{user_compile_flags}"],
+                        iterate_over = "user_compile_flags",
+                        expand_if_available = "user_compile_flags",
+                    ),
+                ],
+            ),
+        ],
+    )
+
+    # obtained with `docker run docker.io/cilium/image-compilers:ac609e4c5a77e9c4c92e096b8c96a49b78d5c869 bash -c 'echo | $(aarch64-linux-gnu-gcc -print-prog-name=cc1plus) -v'`
+    cxx_builtin_include_directories = [
+        "/usr/aarch64-linux-gnu/include/c++/9",
+        "/usr/aarch64-linux-gnu/include/c++/9/aarch64-linux-gnu",
+        "/usr/aarch64-linux-gnu/include/c++/9/backward",
+        "/usr/lib/gcc-cross/aarch64-linux-gnu/9/include",
+        "/usr/aarch64-linux-gnu/include",
+        "/usr/include",
+    ]
+
+    features = [
+        default_compile_flags_feature,
+        default_link_flags_feature,
+        supports_dynamic_linker_feature,
+        supports_pic_feature,
+        feature(name = "opt"),
+        feature(name = "dbg"),
+        user_compile_flags_feature,
+        sysroot_feature,
+        unfiltered_compile_flags_feature,
+    ]
+
+    tool_paths = [
+        tool_path(
+            name = "gcc",
+            path = "/usr/bin/aarch64-linux-gnu-gcc",
+        ),
+        tool_path(
+            name = "ld",
+            path = "/usr/bin/aarch64-linux-gnu-ld",
+        ),
+        tool_path(
+            name = "ar",
+            path = "/usr/bin/aarch64-linux-gnu-ar",
+        ),
+        tool_path(
+            name = "cpp",
+            path = "/usr/bin/aarch64-linux-gnu-cpp",
+        ),
+        tool_path(
+            name = "gcov",
+            path = "/usr/bin/aarch64-linux-gnu-gcov",
+        ),
+        tool_path(
+            name = "nm",
+            path = "/usr/bin/aarch64-linux-gnu-nm",
+        ),
+        tool_path(
+            name = "objcopy",
+            path = "/usr/bin/aarch64-linux-gnu-objcopy",
+        ),
+        tool_path(
+            name = "objdump",
+            path = "/usr/bin/aarch64-linux-gnu-objdump",
+        ),
+        tool_path(
+            name = "strip",
+            path = "/usr/bin/aarch64-linux-gnu-strip",
+        ),
+    ]
+
+    #out = ctx.actions.declare_file(ctx.label.name)
```

Comment:
I haven't quite understood what this was for, but if things work without this, perhaps it should be removed?

Reply:
All commented lines were not needed. I also removed the `-fstack-protector` C(XX)FLAG which doesn't seem to be needed.